<ui version="4.0" >
 <class>StartLevelWidget</class>
 <widget class="QWidget" name="StartLevelWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Start level (dB)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="spin::gui::DecibelLineEdit" name="speechStartLine" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QCheckBox" name="speechCheck" >
     <property name="text" >
      <string>Speech</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="spin::gui::DecibelLineEdit" name="noiseStartLine" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QCheckBox" name="noiseCheck" >
     <property name="text" >
      <string>Noise</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="startNoiseAtLbl" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Noise at:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="showStartPointLbl" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Click to adjust</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>spin::gui::DecibelLineEdit</class>
   <extends>QLineEdit</extends>
   <header>decibellineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
